### PR TITLE
Update community guidelines

### PR DESCRIPTION
**Pull request by rschamp**
### Changes:

Update the content of the Community Guidelines. Changes (and future updates) outlined here https://trello.com/c/Jblclhvv/20-update-community-guidelines-phase-1

### Test Coverage:

The content of the page should match the document linked above.

### CODE REVIEW

**Comment by apple502j** [comment]
Perhaps remove curly quotes and replace with '?

File: src/views/guidelines/guidelines.jsx [hunk 1]
```diff
@@ -16,27 +16,28 @@ const Guidelines = () => (
             }
         >
             <p>
-                <FormattedMessage
-                    className="intro"
-                    id="guidelines.header"
-                />
+                <FormattedMessage id="guidelines.header1" />&nbsp;
+                <strong><FormattedMessage id="guidelines.header2" /></strong>&nbsp;
```
Comment (chrisgarrity) [inline src/views/guidelines/guidelines.jsx:20]: Having these as separate phrases is actually helpful. When they're all one object and we change a small part the whole thing has to be retranslated. If we change one of the sentences the others stay translated.

**Review by chrisgarrity — APPROVED** [review]
Looks good.

**Comment by chrisgarrity** [comment]
@apple502j I just verified that we have the translation checks for quotes enabled, and Transifex prevents you from saving a translation that uses regular quotes. So I think we can start allowing curly quotes where they make sense.

File: src/views/guidelines/l10n.json [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+  "guidelines.header3": "Help keep Scratch a welcoming, supportive, and creative space for all by following these Community Guidelines:",
+  "guidelines.respectheader": "Treat everyone with respect.",
+  "guidelines.respectbody": "Scratchers have diverse backgrounds, interests, identities, and experiences. Everyone on Scratch is encouraged to share things that excite them and are important to them—we hope that you find ways to celebrate your own identity on Scratch, and allow others to do the same. It’s never OK to attack a person or group’s identity or to be unkind to someone about their background or interests.",
+  "guidelines.privacyheader": "Be safe: keep personal and contact information private.",
+  "guidelines.privacybody": "For safety reasons, don't give out any information that could be used for private communication, in person or online. This includes sharing real last names, phone numbers, addresses, hometowns, school names, email addresses, usernames or links to social media sites, video chatting applications, or websites with private chat functionality.",
```
Comment (apple502j) [inline src/views/guidelines/l10n.json:9]: This line is using singlequote. Maybe fix that?

**Pull request merged**